# -----------------------------------------------------------------------------------
# Dynamic Movement Primitive (DMP)
# related parameters used in dmp_parameters.cpp
# -----------------------------------------------------------------------------------

dmp:
  package_name: dmp_motion_generation
  
  library_base_directory_name: library
  
  library_directory_name: dmp_lib
  data_directory_name: dmp_data
  debug_directory_name: dmp_debug

